src/cc2olx/cli.py

```diff
@@ -78,4 +105,12 @@ def parse_args(args=None):
         type=link_source_validator,
         help="The relative links source in the format '<scheme>://<netloc>', e.g. 'https://example.com'",
     )
+    parser.add_argument(
+        "-c",
+        "--content_types_with_custom_blocks",
+        action=AppendIfAllowedAction,
+        default=[],
+        choices=list(SupportedCustomBlockContentType.__members__.values()),
```

Comment:
Is this the same as:

```suggestion
        choices=SupportedCustomBlockContentType,
```

?

Reply:
Ah, okay. But you can still just make it `choice=list(SupportedCustomBlockContentType)`, instead of calling `__members__.values()`, right?

Reply:
Er... or maybe `choices=[str(cb) for cb in SupportedCustomBlockContentType],` to make the output more readable.

Reply:
Yes, I didn't know that `choice=list(SupportedCustomBlockContentType)` is a correct expression. Refactored. Thanks!